build/images/ovs/build.sh

```diff
@@ -77,7 +77,7 @@ done
 # push the "cache image" to the registry. This functionality is not supported
 # with the default docker driver.
 # See https://docs.docker.com/build/cache/backends/registry/
-if $PUSH && ! check_docker_build_driver "docker-container"; then
+if $PUSH && [ "$DISTRO" != "windows" ] && ! check_docker_build_driver "docker-container"; then
```

Comment:
May I know why adding windows condition here?

Reply:
I want to use ovs/build.sh to support buiding Windows OVS images too, and from my test, buildx with driver "docker-container"  was not mandatory for Windows. So I bypass Windows case in this check